src/main/java/rodain/source/ui/ComputeDirectorySize.java

```diff
@@ -1,46 +1,67 @@
-package source.ui;
+package rodain.source.ui;
 
-import java.nio.file.*;
+import java.nio.file.Path;
 import java.nio.file.attribute.BasicFileAttributes;
 import java.util.Observable;
 
-import org.slf4j.LoggerFactory;
-import utils.TreeVisitor;
+import rodain.utils.TreeVisitor;
 
 /**
  * Created by adrapereira on 24-09-2015.
  */
 public class ComputeDirectorySize extends Observable implements TreeVisitor {
-    private static final org.slf4j.Logger log = LoggerFactory.getLogger(ComputeDirectorySize.class.getName());
-    private final int UPDATEFREQUENCY = 500; //in milliseconds
+    private static final int UPDATE_FREQUENCY = 500; //in milliseconds
```

Comment:
This UPDATE_FREQUENCY = 500 appears in several classes.
You can create a static class for constants or put this information on a configuration properties file.